knowledge_graphs/knowledge_graph_validator.py

```
@@ -0,0 +1,310 @@
+                    "name": op,
+                    "status": "safe_pattern"
+                })
+    
+    def _is_standard_library(self, module_name: str) -> bool:
+        """Check if a module is part of Python standard library."""
+        standard_libs = {
+            'os', 'sys', 'json', 'time', 'datetime', 'collections', 'itertools',
+            'functools', 'pathlib', 'urllib', 'http', 'logging', 'argparse',
+            'subprocess', 'threading', 'asyncio', 'typing', 're', 'math',
+            'random', 'sqlite3', 'csv', 'xml', 'html', 'email', 'base64',
+            'hashlib', 'hmac', 'secrets', 'ssl', 'socket', 'tempfile'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)